api/service/explorer/service.go

```diff
@@ -300,7 +289,16 @@ func (s *Service) GetExplorerAddress(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	result = data.Address
+	db := s.storage.GetDB()
+	bytes, err := db.Get([]byte(key))
+	if err != nil {
+		ctxerror.Warn(utils.GetLogger(), err, "cannot read address from db", "id", id)
```

Comment:
why different usage here? and line 299?

Reply:
i kept the same logic of the original code. just move the logic of getbalance at the front.